pkg/collector/py/check.go

```diff
@@ -50,12 +52,19 @@ func NewPythonCheck(name string, class *python.PyObject) *PythonCheck {
 		interval:     check.DefaultCheckInterval,
 		lastWarnings: []error{},
 	}
+	pyCheck.stopped.Store(false)
```

Comment:
With the solution in this PR, I don't think the changes here (`pkg/collector/py/check.go`) really make sense. These checks should be common to all types of checks IMO, and should be done from the runner before calling `check.Run()`

Reply:
This should be fine, it was a way to at least get rid of the error, given the numerous race conditions that were getting sorted with the other PRs.